sdks/typescript/src/v1/examples/on_success/workflow.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+import { hatchet } from '../hatchet-client';
+
+// ❓ On Success Task
+export const onSuccess = hatchet.task({
+  name: 'run-on-success',
+  // 👀 onSuccess handler will run if the fn task succeeds
+  onSuccess: (_, ctx) => {
+    console.log('onSuccess for run:', ctx.workflowRunId());
+    return {
+      'on-success': 'success',
+    };
+  },
```

Comment:
I wasn't exposing `on_success` or `on_failure` for the single task thing either. Do you think we should do this? I sorta think no, for a similar reason to the DAG stuff

Reply:
i dont think there's much harm in exposing it for now, i think its a good goal to make task <> workflow as close as possible